array_api_tests/pytest_helpers.py

```diff
@@ -420,6 +421,35 @@ def assert_fill(
         assert xp.all(xp.equal(out, xp.asarray(fill_value, dtype=dtype))), msg
 
 
+def _real_float_strict_equals(out: Array, expected: Array) -> bool:
+    nan_mask = xp.isnan(out)
+    if not xp.all(nan_mask == xp.isnan(expected)):
+        return False
+    ignore_mask = nan_mask
+
+    # Test sign of zeroes if xp.signbit() available, otherwise ignore as it's
+    # not that big of a deal for the perf costs.
+    if api_version >= "2023.12" and hasattr(_xp, "signbit"):
```

Comment:
Why do we need the version check here? If the library has signbit then we can use it.